numba/unicode.py

```diff
@@ -541,6 +541,72 @@ def find_impl(a, b):
         return find_impl
 
 
+@overload_method(types.UnicodeType, 'rfind')
+def unicode_rfind(s, substr, start=None, end=None):
+    """Implements str.rfind()"""
+    def unicode_rfind_check_type(ty, name):
+        """
+        Check object belongs to one of specific types
```

Comment:
While this isn't made too explicit in the docs, we do use the Numpy docstring style guide:

https://numpydoc.readthedocs.io/en/latest/format.html